fluss-flink/fluss-flink-common/src/test/java/com/alibaba/fluss/flink/sink/FlussSinkITCase.java

```diff
@@ -0,0 +1,352 @@
+import com.alibaba.fluss.flink.sink.serializer.RowDataSerializationSchema;
+import com.alibaba.fluss.flink.source.testutils.FlinkTestBase;
+import com.alibaba.fluss.flink.utils.FlussRowToFlinkRowConverter;
+import com.alibaba.fluss.metadata.Schema;
+import com.alibaba.fluss.metadata.TableBucket;
+import com.alibaba.fluss.metadata.TableDescriptor;
+import com.alibaba.fluss.metadata.TablePath;
+import com.alibaba.fluss.row.BinaryString;
+import com.alibaba.fluss.row.GenericRow;
+import com.alibaba.fluss.row.InternalRow;
+import com.alibaba.fluss.types.DataTypes;
+
+import org.apache.flink.streaming.api.datastream.DataStream;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.table.data.RowData;
+import org.apache.flink.table.types.logical.BigIntType;
+import org.apache.flink.table.types.logical.IntType;
+import org.apache.flink.table.types.logical.LogicalType;
+import org.apache.flink.table.types.logical.RowType;
+import org.apache.flink.table.types.logical.VarCharType;
+import org.apache.flink.types.RowKind;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.Serializable;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import static com.alibaba.fluss.flink.utils.FlinkConversions.toFlinkRowKind;
+import static com.alibaba.fluss.testutils.DataTestUtils.row;
+
+/** Integration tests for the Fluss sink connector in Flink. */
+public class FlussSinkITCase extends FlinkTestBase {
+    private static final Schema pkSchema =
+            Schema.newBuilder()
+                    .column("orderId", DataTypes.BIGINT())
+                    .column("itemId", DataTypes.BIGINT())
+                    .column("amount", DataTypes.INT())
+                    .column("address", DataTypes.STRING())
+                    .primaryKey("orderId")
+                    .build();
+
+    private static StreamExecutionEnvironment env;
+
+    private static TableDescriptor pkTableDescriptor;
+
+    private static String bootstrapServers;
+
+    private static String pkTableName = "orders_test_pk";
+
+    private static TablePath ordersPKTablePath;
+
+    @BeforeEach
+    public void setup() throws Exception {
+        bootstrapServers = conn.getConfiguration().get(ConfigOptions.BOOTSTRAP_SERVERS).get(0);
+
+        pkTableDescriptor =
+                TableDescriptor.builder().schema(pkSchema).distributedBy(1, "orderId").build();
+
+        TablePath pkTablePath = TablePath.of(DEFAULT_DB, pkTableName);
+
+        createTable(pkTablePath, pkTableDescriptor);
+
+        this.ordersPKTablePath = new TablePath(DEFAULT_DB, pkTableName);
+
+        this.bootstrapServers = conn.getConfiguration().get(ConfigOptions.BOOTSTRAP_SERVERS).get(0);
+
+        env = StreamExecutionEnvironment.getExecutionEnvironment();
+        env.setParallelism(2);
```

Comment:
Could you rebase the branch to the latest `main`, and take `com.alibaba.fluss.flink.sink.FlinkTableSinkITCase` as an example about how to initialize `env`? We changed the way of env initialization in the recent days to make the tests more stable.